microsoft-edge/devtools-guide-chromium/experimental-features.md

```diff
@@ -0,0 +1,69 @@
+---
+description: The latest experimental features in Edge DevTools
+title: Experimental features
+author: hxlnt
+ms.author: raweil
+ms.date: 06/15/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, experiment
+---
+
+# Experimental features
+
+The Edge DevTools periodically offers experimental features that are released while still in development. By enabling an experiment, you are able to test and [provide early feedback](#Providing-feedback-on-experimental-features) on these features before they are released broadly.
+
+Experiments are available in all channels of Edge. Use the Edge Canary channel to get the very latest experiments.
+
+## How to enable experiments
+
+To enable an experiment, open DevTools by pressing `Control+Shift+I` (Windows) or `Command+Option+I` (macOS). Then, open Settings by clicking the gear icon at the top right of DevTools or by pressing `F1`.
+
+On the left side of the Settings panel, there is a tab called "Experiments." Click this tab to see all available experiments. Select the checkboxes next to experiments you would like to try, then close and reopen the DevTools.
+
+![List of experiments in DevTools Settings](./media/experiments-devtools.png)
+
+Note that some experimental features are works in progress and may cause performance issues. You can remove an experiment by unchecking the box next to that experiment and reloading the DevTools as before.
+
+## Current experimental features
+
+The following sections outline some of the experimental features available in Edge now.
+
+### "Enable new CSS grid debugging features"
+
+This feature offers improved on-page visualizations for websites using CSS grid layouts.
+
+![CSS grid debugging feature](./media/experiments-grid.png)
+
+This experiment is available in Edge 85 and later.
+
+### "Enable support to move tabs between panels"
+
+Normally, tools such as Elements and Network can only be open in the main (top) panel of DevTools. Likewise, tools such as 3D View and Issues can only be open in the drawer (bottom) panel of DevTools. However, when this experiment is enabled, you are able to move tools between the top and bottoms panels by right-clicking the tab and choosing "Move to top" or "Move to bottom." This allows you to better customize your DevTools layout.
+
+![Moving tabs between panels](./media/experiments-move-panels.png)
+
+The bottom panel can be hidden and shown by pressing ``Ctrl + ` ``.
+
+This experiment is available in Edge 85 and later.
+
+### "Enable webhint"
```

Comment:
Could we expand on what this experiment is named in the tools? "Enable webhint" is ambiguous...would be great to communicate where webhint would surface, like "Enable webhint suggestions in the Issues panel".

Reply:
Initially, I had more "friendly" titles here, but I realized that it's hard to locate the experiment in Settings unless you know exactly what it's called because there are so many experiments. The quotemarks around the title were meant to denote that it's the name of the setting rather than a description of it, enabling you to find it easily.